nautobot/ipam/tests/test_filters.py

```diff
@@ -268,37 +269,43 @@ def setUpTestData(cls):
             rir=rirs[0],
             tenant=tenants[0],
             date_added="2020-01-01",
-        ),
+        )
         Aggregate.objects.create(
             prefix="10.2.0.0/16",
             rir=rirs[0],
             tenant=tenants[1],
             date_added="2020-01-02",
-        ),
+        )
         Aggregate.objects.create(
             prefix="10.3.0.0/16",
             rir=rirs[1],
             tenant=tenants[2],
             date_added="2020-01-03",
-        ),
+        )
         Aggregate.objects.create(
             prefix="2001:db8:1::/48",
             rir=rirs[1],
             tenant=tenants[0],
             date_added="2020-01-04",
-        ),
+        )
         Aggregate.objects.create(
             prefix="2001:db8:3::/48",
             rir=rirs[2],
             tenant=tenants[2],
             date_added="2020-01-06",
-        ),
+        )
         Aggregate.objects.create(
             prefix="2001:db8:2::/48",
             rir=rirs[2],
             tenant=tenants[1],
             date_added="2020-01-05",
-        ),
+        )
+
+    def test_search(self):
+        Aggregate.objects.create(prefix="10.150.255.0/31", rir=self.rirs[0])
+        Aggregate.objects.create(prefix="10.150.255.2/31", rir=self.rirs[0])
+        params = {"q": "10.150.255.2/31"}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
```

Comment:
Mind also adding a search for `"10.150.255.0"` and/or `"10.150.255.2"` to demonstrate the expected behavior after fixing #286? Same request on the other test case below.